### DISCUSSION TITLE
What is the proper/preferred way to update PGMA to the latest

### DISCUSSION THREAD

**Discussion by jfreiman** [CodyBerenson/PGMA-Modernized#146] [Q&A]
I'm curious as to the best way to update the ..\Plug-ins\ folder with the latest build of PGMA

- Is it best to extract the contents of the Zip file over the existing installed build? (and replacing the old files)
- Or, is it best to delete the contents of the ..\Plug-ins\ folder and then extract the Zip into the clean folder?

I couldn't locate this information in the included PDF.

**Comment by JPH71** [comment]
Delete and replace... the you are sure

**Comment by JPH71** [comment]
However all your preferences will be rest in this case

**Comment by jfreiman** [comment]
Ok, good to know.
I didn't like having to verify/modify the settings for each Agent - that makes sense so any issues fixed are sure to be rooted out by the new build/code.
Thank you

**Comment by JPH71** [comment]
I have loads of little niggling issues to sort out this week... which I
should have finished this weekend....
so as soon as you find something not behaving right or want an improvement
let me know....
Also raise them as issues in the repository, so i can tick them off as I
solve them.


Jason xxx
